src/sorcha/ephemeris/simulation_parsing.py

```diff
@@ -47,6 +58,17 @@ def parse_orbit_row(row, epochMJD_TDB, ephem, sun_dict, gm_sun):
                 epochMJD_TDB - (row["ma"] * np.pi / 180.0) * np.sqrt(row["a"] ** 3 / gm_sun),
                 epochMJD_TDB,
             )
+        elif orbit_format == "BKEP":
+            ecx, ecy, ecz, dx, dy, dz = universal_cartesian(
+                gm_total,
+                row["a"] * (1 - row["e"]),
+                row["e"],
+                row["inc"] * np.pi / 180.0,
+                row["node"] * np.pi / 180.0,
+                row["argPeri"] * np.pi / 180.0,
+                epochMJD_TDB - (row["ma"] * np.pi / 180.0) * np.sqrt(row["a"] ** 3 / gm_total),
+                epochMJD_TDB,
+            )
         else:
             raise ValueError("Provided orbit format not supported.")
```

Comment:
While we're here can we get this error outputted to the log.

Reply:
This is now caught in a try/except statement in the function that call this one. If the ValueError is raised, the `except` section will log it to the .log and .err files and then call sys.exit.